sdk/appconfiguration/Azure.Data.AppConfiguration/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## 1.0.0-preview.6
 
-- Bugfixes
+- Bugfixes: [#8920](https://github.com/Azure/azure-sdk-for-net/issues/8920)
```

Comment:
Is there a precedent for linking the issue?  I had thought we summarized them in sentences.

Reply:
@AlexGhiondea asked me to add exact bug number